### PR TITLE
Log when pinging

### PR DESCRIPTION
**Pull request by kihehs**
This helps us determine if the token is correct.

### CODE REVIEW

**Review by Tpt — APPROVED** [review]

File: src/registry/artifactory.rs [hunk 1]
```diff
@@ -75,6 +75,11 @@ impl Artifactory {
         };
 
         let status = response.status();
+
+        if !status.is_success() {
+            tracing::info!(response_header=?response.headers());
```
Comment (Tpt) [inline src/registry/artifactory.rs:80]: debug maybe?

Comment (alexespencer) [reply src/registry/artifactory.rs:80]: If we use debug we may need to override RUST_LOG env var in CI - hopefully this should be possible?

Comment (ntrinquier) [reply src/registry/artifactory.rs:80]: You might want to consider the [debug string](https://github.com/seanmonstar/reqwest/blob/3abcc7c4f537c16ad9937f8cc60fb23cb506ac85/src/async_impl/response.rs#L389-L397) too:

```suggestion
            tracing::debug!("artifactory response: {:?}", response);
```

Comment (Tpt) [reply src/registry/artifactory.rs:80]: Yes. You have an example here: https://github.com/helsing-ai/buffrs/blob/3120b1a7bb16d8fd47cfbe98ced5dcfa9635df41/.github/workflows/ci.yml#L34

Comment (kihehs) [reply src/registry/artifactory.rs:80]: We're trying to fix a bug that only appears in CI, and I'm not sure how to override RUST_LOG there.

Comment (kihehs) [reply src/registry/artifactory.rs]: Regarding `debug!`, I don't see my debug logs when I run this. I seem to remember a lot of custom initialization being done to the:

```bash
RUST_LOG=debug cargo run -- login --registry https://my-registry...
export RUST_LOG=debug; cargo run -- login --registry https://my-registry...
```

(This doesn't change with `RUST_BACKTRACE=1`)

**Comment by kihehs** [comment]
Closing because another merge breaks this PR pretty badly.

**Pull request closed without merging**